### ISSUE TITLE
AutoHeight too short for Google Colab

### ISSUE THREAD

**Issue by paddymul** [buckaroo-data/buckaroo#256]
### Checks

- [X] I have checked that this issue has not already been reported.

- [X] I have confirmed this bug exists on the [latest version](https://github.com/paddymul/buckaroo) of Buckaroo.

### What type of jupyter notebook were you using (VSCode notebook, google colab, Jupyter Lab, Jupyter notebook).  Select multiple if you can reproduce this in multiple environments.  If other, please add to description.

Google Colab

### Reproducible example

```python
import pandas as pd
import buckaroo
num_df = pd.DataFrame(
    {'a':[111_111,  77_777, 777_777, 1_000_000, 2_111_111, 1_235_999],
     'b':[111_111, 555_555,       0,    28_123,   482_388,     5_666]})
num_df
```

### Issue description

<img width="721" alt="Screenshot 2024-03-15 at 8 37 43 AM" src="https://github.com/paddymul/buckaroo/assets/40453/7a468186-5fd7-414f-8be8-1bbbd424c869">

### Expected behavior

the display should be taller

### Installed versions

<details>
```
Replace this line with the output of buckaroo.debug_packages(). Leave the backticks in place.
```
</details>

**Comment by paddymul** [comment]
<img width="826" alt="Screenshot 2024-03-22 at 9 57 17 AM" src="https://github.com/paddymul/buckaroo/assets/40453/ce20fcb6-c7b3-402c-9b45-10be81fae7d1">

```python
buckaroo.BuckarooWidget(num_df,component_config={'dfvHeight':250})
```

**Comment by paddymul** [comment]
Fixed as of `0.6.9` 
<img width="982" alt="Screenshot 2024-03-22 at 3 34 34 PM" src="https://github.com/paddymul/buckaroo/assets/40453/6c42f8b7-02dc-41f4-93f1-d706ba56c86c">
